library/src/main/java/com/anjlab/android/iab/v3/BillingProcessor.java

```diff
@@ -296,10 +297,14 @@ private boolean loadPurchasesByType(String type, BillingCache cacheStorage)
 		return false;
 	}
 
+	/**
+	 * Attempt to fetch purchases from the server and update our cache if successful
+	 * @return {@code true} if retrieval occurs, {@code false} otherwise
+	 * todo check if loading should return true if response code isn't successful
+	 */
 	public boolean loadOwnedPurchasesFromGoogle()
```

Comment:
Currently, the load methods return true so long as a response is received, regardless of whether the response was successful or not. I also removed the first `isInitialized` call as it is called in both load methods. It therefore doesn't need to be verified a third time beforehand